radar-spring-auth/src/main/kotlin/radar/spring/auth/managementportal/ManagementPortalAuthorization.kt

```diff
@@ -1,12 +1,23 @@
 package radar.spring.auth.managementportal
 
+import kotlinx.coroutines.runBlocking
+import org.radarbase.auth.authorization.EntityDetails
+import org.radarbase.auth.authorization.EntityRelationService
+import org.radarbase.auth.authorization.MPAuthorizationOracle
 import org.radarbase.auth.authorization.Permission
 import org.radarbase.auth.token.RadarToken
 import org.slf4j.LoggerFactory
 import radar.spring.auth.common.Authorization
 import radar.spring.auth.common.PermissionOn
 
-class ManagementPortalAuthorization : Authorization<RadarToken> {
+class ManagementPortalAuthorization() : Authorization<RadarToken> {
+    private val relationService =
+        object : EntityRelationService {
+            override suspend fun findOrganizationOfProject(project: String): String? {
+                return null
```

Comment:
is this intended?

Reply:
I've updated it to use the `main` project by default. I thought since we don't use organizations in appserver, we don't need to explicitly define this for now?